app/src/main/java/com/github/muellerma/mute_reminder/ForegroundService.kt

```diff
@@ -75,16 +78,26 @@ class ForegroundService : Service() {
             shouldNotify -> {
                 Log.d(TAG, "Should notify, show notification")
 
+                //create pendingIntent, click to send mute action
+                val intent = Intent(this, ForegroundService::class.java)
+                intent.action = "mute_media"
+                val pendingIntent = PendingIntent.getService(this, 0, intent,
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                        PendingIntent.FLAG_IMMUTABLE or PendingIntent.FLAG_UPDATE_CURRENT
+                    } else { PendingIntent.FLAG_UPDATE_CURRENT }
```

Comment:
Can you add the code I've done for an Android app to avoid checking for the version here?

https://github.com/openhab/openhab-android/blob/main/mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt#L557-L567